client/vault/smoke_test.py

```
@@ -0,0 +1,1126 @@
+            account_id=AccountId(config.allocator_account),
+            secret_key=config.allocator_secret_key,
+        )
+        allocator_client = VaultClient.new_single_key(
+            rpc_url=config.rpc_url,
+            vault=AccountId(config.vault_account),
+            credential=allocator_cred,
+            config=client_config,
+        )
+        print(f"  ✓ Allocator client created for: {config.allocator_account}")
+    else:
+        print(f"  ⚠ Allocator client NOT created (missing credentials)")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)